packages/api/src/utils.ts

```diff
@@ -23,3 +24,16 @@ export const isDiscordAdmin = async (user: Session["user"]) => {
   )) as APIGuildMember;
   return guildMember.roles.includes(DISCORD_ADMIN_ROLE_ID);
 };
+
+const auth = new google.auth.JWT(
+  env.GOOGLE_CLIENT_EMAIL,
+  undefined,
+  env.GOOGLE_PRIVATE_KEY,
+  ["https://www.googleapis.com/auth/calendar"],
+  "dylan@knighthacks.org",
```

Comment:
it may be beneficial to make the email a const at the top of the file for later on (line 33)